common/types/json_struct_test.go

```diff
@@ -64,6 +66,25 @@ func TestJsonStruct_ConvertToNative_Json(t *testing.T) {
 	}
 }
 
+func TestJsonStruct_ConvertToNative_Any(t *testing.T) {
+	structVal := &structpb.Struct{
+		Fields:map[string]*structpb.Value{
+			"first":  {Kind:&structpb.Value_StringValue{"hello"}},
+			"second": {Kind:&structpb.Value_NumberValue{1}}}}
+	mapVal := NewJsonStruct(structVal)
+	anyVal, err := mapVal.ConvertToNative(reflect.TypeOf(&any.Any{}))
+	if err != nil {
+		t.Error(err)
+	}
+	unpackedAny := ptypes.DynamicAny{}
+	if ptypes.UnmarshalAny(anyVal.(*any.Any), &unpackedAny) != nil {
+		NewErr("Failed to unmarshal any")
```

Comment:
Please use `t.Error()` instead. The `NewErr` call is used for creating an error value that may be processed during  expression evaluation.

Reply:
done.